packages/grid/docker-compose.build.yml

```diff
@@ -5,9 +5,13 @@ services:
       context: ${RELATIVE_PATH}./frontend
       dockerfile: frontend.dockerfile
       target: "${FRONTEND_TARGET:-grid-ui-development}"
+    profiles:
+      - frontend
 
   backend:
     build:
       context: ${RELATIVE_PATH}../
       dockerfile: ./grid/backend/backend.dockerfile
       target: "backend"
+    profiles:
+      - backend
```

Comment:
Q: We might need to add a section for worker, for local build of worker container with current changes locally